index.html

```diff
@@ -18,16 +21,178 @@
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
+                </div>
+            </div>
+            <!-- end card4 -->
+            <!-- start card5 -->
+            <div class="project-cards project5">
+                <div class="pjimgs">
+                    <img src="./img/pj5.svg" alt="pj5"/>
+                </div>
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
+                </div>
+            </div>
+            <!-- end card5 -->
+            <!-- start card6 -->
+            <div class="project-cards project6">
+                <div class="pjimgs">
+                    <img src="./img/pj6.svg" alt="pj6"/>
+                </div>
+                <div class="pjcontent">
+                    <p class="pjname">Project name goes here</p>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javasccript</li>
+                    </ul>
+                    <button class="pjbtn"><a href="#"><span>See this Project</span> <img src="./img/union.svg" alt="rightarrow"/></a></button>
+                </div>
+            </div>
+            <!-- end card6 -->
+        </div>
+        <!-- end project card -->
+    </section>
+    <!-- End WORK SECTION -->
+
+    <!-- Start ABOUT SECTION -->
+    <section class="about">
+        <div class="aboutintro">
+            <!-- start title -->
+            <div class="title">
+                <h1>About</h1>
+            </div>
+            <!-- end title -->
+            <div class="bio">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+            </div>
+
+            <button type="button" class="cvbtn"><a href="#"><span>Get My CV</span><img src="./img/downloadicon.svg" alt="icon"/></a></button>
+        </div>  
+        
+        <div class="section-card">
+            <ul class="card-list">
+                <li class="cards card1">
+                    Languages
+                    <p>
+                        • JavaScript (ES5/ES6)<br/>
+                        • Ruby<br/>
+                        • HTML<br/>
+                        • CSS
+                    </p>
```

Comment:
Please make this item a list item inside the a `ul` tag to make your code more semantic